.github/workflows/continuous-integration-workflow.yaml

```diff
@@ -22,7 +22,7 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install -e ".[develop]"
-        pip install git+https://github.com/NREL/electrolyzer.git

```

Comment:
Allows us to use the updated inputs to the electrolyzer in the regression tests.